frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/test-expand.scenario.ts

```diff
@@ -0,0 +1,76 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { execSync } from 'child_process';
+import { clickKebabAction } from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { testPVC, testDeployment } from '../../mocks/expand-test-mocks';
+import { requestSizeValue } from '../../views/add-capacity.view';
+import {
+  goToPersistentVolumeClaims,
+  expandButton,
+  createNewPersistentVolumeClaim,
+  expandSizeOption,
+  capacityDropdown,
+} from '../../views/pvc.view';
+import { SIZE_UNITS } from '../../utils/consts';
+
+const expandValue = String(Number(testPVC.size) + 1);
+
+const createDeployment = () =>
+  execSync(`echo '${JSON.stringify(testDeployment)}' | kubectl create -f -`);
+
+describe('Tests Expand flow for PVC', () => {
+  beforeAll(async () => {
+    // Create a PVC
+    await createNewPersistentVolumeClaim(testPVC, true, createDeployment);
+    // Wait for it to bind
+    await goToPersistentVolumeClaims();
```

Comment:
You should also check that the new PVC that is created should be in Bound State before expanding it as I don't see that check in `goToPersistentVolumeClaims`

Reply:
Sorry for the misleading comment, but the check is done by `createNewPersisenteVolumeClaim` the second parameter `true` allows us to either not/wait for a PVC to bind.

Reply:
Ack, please update the comment then as it can be confusing.

Reply:
`goToPersistentVolumeClaims` is already being called inside `createNewPersistentVolumeClaim`. Do you need to call it again here?

Reply:
Yes, I need to call it. If you expand a PVC the request takes you to the detail page of that particular PVC.

Reply:
Ack